assignment.js

```diff
@@ -104,10 +122,23 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
-
+let oneToHundredArray = []
+
```

Comment:
🛠 Read the comments below and update your code accordingly

@vasynizami good use of the `for` loop here and the `if` statement is structurally sounds. 

While your implementation is close but not quite there yet.  The following changes need to be made:

1. Move your `if..else` statement **inside** of your `for` loop 

2. Check the order of your clauses within your `if..else if` statement- make sure your program prints "fizzbuzz" for numbers (like **15**) that are divisible by both 3 and 5.